src/components/user-settings.tsx

```diff
@@ -162,11 +160,25 @@ export default function UserSettings() {
 	} );
 
 	const onRemoveSnapshots = useCallback( async () => {
-		if ( snapshots?.length === 0 ) {
+		if ( allSites?.length === 0 ) {
 			return;
 		}
-		await deleteAllSnapshots( snapshots );
-	}, [ snapshots, deleteAllSnapshots ] );
+
+		const CANCEL_BUTTON_INDEX = 0;
+		const DELETE_BUTTON_INDEX = 1;
+
+		const { response } = await getIpcApi().showMessageBox( {
+			type: 'warning',
+			message: __( 'Delete all demo sites' ),
+			detail: __( "Any changes you've made to your demo sites will be lost." ),
+			buttons: [ __( 'Cancel' ), __( 'Delete all' ) ],
```

Comment:
should the `Delete all` be set to blue color similar to the native dialogue that we have for deleting a single site?

<img width="252" alt="Screenshot 2024-05-16 at 10 53 54 AM" src="https://github.com/Automattic/studio/assets/25575134/bd25d565-3cf6-417a-8db5-329e802d21a8">

Reply:
> should the Delete all be set to blue color similar to the native dialogue that we have for deleting a single site?

I'm not sure I can do that @katinthehatsite . In my machine, there is no styling at all. Maybe can you help me here somehow?

Reply:
@kozer could we make the order of the buttons consistent for both confirmation dialogs?

Also, could we use the text from the existing dialog but modify it to account for multiple demo sites being removed, eg.?

> All demo sites' databases that exist for your WordPress.com account, along with all posts, pages, comments, and media, will be lost.